### PR TITLE
Feat/map on itinerary

### PR DESCRIPTION
**Pull request by derjubu**
Part of #17 

- Create ItineraryMap component
- Show marker in map for each destination on Itinerary page

### CODE REVIEW

File: src/app/components/ItineraryMap/ItineraryMap.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }, []);
+
+  return (
+    <div>
+      <MapContainer ref={mapContainer} />
+    </div>
+  );
+}
+const MapContainer = styled.div`
+  height: 400px;
```
Comment (janhendrikstruve) [inline src/app/components/ItineraryMap/ItineraryMap.tsx:51]: Is a fixed pixel height the best solution here? Really don't know, it was just something i've noticed.

**Review by janhendrikstruve — COMMENTED** [review]
Looks very good, nothing to complain, just noticed one thing that maybe could be an issue in terms of responsiveness.

File: src/app/pages/Itinerary/Itinerary.tsx [hunk 2]
```diff
@@ -2,11 +2,25 @@ import React from 'react';
...
+
+  {
+    Itinerary.map((destination: DestinationType) =>
+      locationsCoordinates.push(
+        destination.location.geometry.coordinates as LngLatLike
+      )
+    );
+  }
+
+  console.log(locationsCoordinates);
```
Comment (PReichetanz) [inline src/app/pages/Itinerary/Itinerary.tsx]: I guess you can delete this and the empty lines around.

**Review by PReichetanz — COMMENTED** [review]
Looks good to me!

File: src/app/pages/Itinerary/Itinerary.tsx (hunk 2 above)
Comment (derjubu) [reply src/app/pages/Itinerary/Itinerary.tsx]: Thanks for spotting!

File: src/app/components/ItineraryMap/ItineraryMap.tsx (hunk 1 above)
Comment (derjubu) [reply src/app/components/ItineraryMap/ItineraryMap.tsx:51]: Thanks for spotting. I will leave this for now and will take care of it when I give the whole app a makeover in the last week.

**Pull request merged**